dune-project

```
@@ -85,6 +86,7 @@
   tls-async
   lwt_ssl
   (mdx (and (>= "2.2.1") :with-test))
+  eio_main
```

```suggestion
  (eio_main (>= 0.10))
```
Add a lower bound on this? I am generally in favour of including them for libraries, plus ocaml-ci will check the project works with lower bounds.

---

Thanks! Agreed.